readme.md

```diff
@@ -1,88 +1,183 @@
-You should at least upload the script to your server.
+This is a simple way to start but not recommended because AdBlock and other browser extensions can block the script.
+You should at least upload the script file to your server.
 
-### Webpack/Rollup/Browserify
+### Or from NPM and use with Webpack/Rollup/Browserify
 
 ```bash
 npm i @fingerprintjs/fingerprintjs
+# or
+yarn add @fingerprintjs/fingerprintjs
 ```
 
 ```js
-import * as FPJS from '@fingerprintjs/fingerprintjs';
+import * as FingerprintJS from '@fingerprintjs/fingerprintjs';
 
 (async () => {
-  const fpjs = await FPJS.load();
-  const { visitorId } = await fpjs.get();
-  console.log(visitorId)
+  // It's good to call this when the application starts
+  const fpAgent = await FingerprintJS.load();
+
+  // The FingerprintJS agent is ready. Get a visitor identifier when you'd like to.
+  const result = await fpAgent.get();
+
+  // This is the visitor identifier:
+  const visitorId = result.visitorId;
+  console.log(visitorId);
 })();
 ```
 
-## Version policy
+## Pro version
 
-The library doesn't promise same visitor identifier between any versions
-but tries to keep them same as much as reasonable.
+Upgrade to [Pro version](https://fingerprintjs.com) to get 99.5% accuracy of identification.
 
-The documented JS API follows [Semantic Versioning](https://semver.org).
-Using undocumented features is at you own risk.
+<p align="center">
+  <a href="https://fingerprintjs.com">
+    <img src="resources/pro_screenshot.png" alt="Pro screenshot" width="697px" />
+  </a>
+</p>
 
-## Contribution
+| | Open Source version | Pro version |
+|-----|-------|-------|
+| Browser identification | Basic | Advanced |
+| Anonymous user identification | ❌ | ✅ |
+| Bot detection | ❌ | ✅ |
+| Incognito / Private mode detection | ❌ | ✅ |
+| Geolocation | ❌ | ✅ |
+| Security | ❌ | ✅ |
+| Server API | ❌ | ✅ |
+| Webhooks | ❌ | ✅ |
 
-### Development playground
+Pro result example:
 
-```bash
-# Make sure you have Yarn installed
-yarn install
-yarn playground:start # Add '--port 8765' to change the server port
+```js
+{
+  "requestId": "HFMlljrzKEiZmhUNDx7Z",
+  "visitorId": "kHqPGWS1Mj18sZFsP8Wl",
+  "visitorFound": true,
+  "incognito": false,
+  "bot": undefined,
+  "ip": "192.65.67.131",
+  "browserName": "Chrome",
+  "browserVersion": "85.0.4183",
+  "os": "Mac OS X",
+  "osVersion": "10.15.6",
+  "device": "Other",
+  "ipLocation": {/* ... */}
+}
 ```
 
-Then open http://localhost:8080 in a browser.
-It's reloaded every time the source code is changed.
+🍿 [Live demo](https://fingerprintjs.com/demo)
 
-To build the playground distribution code (e.g. to upload to a static server), run:
+⏱ [How to upgrade from Open Source to Pro in 30 seconds]()
 
-```bash
-yarn playground:build
-```
+📗 [FingerprintJS Pro documentation](https://dev.fingerprintjs.com)
+
+## Open version reference
+
+The library is shipped in various formats:
+
+- Plain JS
```

Comment:
is it a thing? No better name for it?

Reply:
What would you call it? It can be called:

- Simple `<script>` tag
- HTML script
- IIFE
- Global variable 

It’s not correct to call it “CDM”, because it’s not required to load the script from CDN and UMD can be loaded from CDN too.